src/debian/sudoers/Debian-snmp

```diff
@@ -1,3 +1,4 @@
-Debian-snmp     ALL = (root) NOPASSWD: /usr/local/bin/snmp_diskusage.sh
-Debian-snmp     ALL = (root) NOPASSWD: /usr/local/bin/snmp_cephstatus.sh
-Debian-snmp     ALL = (root) NOPASSWD: /usr/local/bin/snmp_cephusage.sh
+Defaults:Debian-snmp !requiretty
```

Comment:
The shellbang /usr/bin/env bash must also be changed in all these scripts. Otherwise it is too easy to set change the environment to gain privilege.